docs/components-upgrade.md

```diff
@@ -82,3 +84,61 @@ If something goes wrong with the upgrade, or if tests are not successful after u
 ### Hotfix "in-place" upgrades
 
 This process will restart tomcat on all the repository nodes and there is no guarantee one node is stopped only after the others have restarted. As a consequence a short service outage needs to be scheduled
+
+## Specifing a different component repository
```

Comment:
I would say that this file is not the best one given that it reference explicitly upgrading components and this topic is more broad

I will move everything under the deployment-guide in the configure-your-deployment section